sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/physical/partitioning.scala

```diff
@@ -90,6 +90,35 @@ case class ClusteredDistribution(
   }
 }
 
+/**
+ * Represents the requirement of distribution on the stateful operator.
```

Comment:
nit: do we need to put "structured streaming" before "stateful operator"?

Reply:
I think "stateful" is already representing the streaming context, but no big deal if we repeat here.